website/docs/docs/build/packages.md

```diff
@@ -157,9 +157,60 @@ packages:
 
 Where `name: 'dbt_utils'` specifies the subfolder of `dbt_packages` that's created for the package source code to be installed within.
 
-### Private packages
+## Private packages
 
-#### SSH Key Method (Command Line only)
+### Native private packages <Lifecycle status='beta'/> 
+
+dbt Cloud supports private packages from [supported](#prerequisites) Git repos leveraging an exisiting [configuration](/docs/cloud/git/git-configuration-in-dbt-cloud) in your environment. Previously, you had to configure a [token](#git-token-method) to retrieve packages from your private repos.  
+
+#### Prerequisites
+
+Native support for private packages is supported on the following services:
+- [Github](/docs/cloud/git/connect-github)
+- [Azure DevOps](/docs/cloud/git/connect-azure-devops)
+
+Support for Gitlab is coming soon. 
+
+To use this feature, you must have a supported Git provider configured in the **Integrations** section of your **Account settings**.
+
+#### Configuration
+
+Use the `private` block in your `packages.yml` or `dependencies.yml` to clone package repos using your existing dbt Cloud Git integration without having to provision an access token or create a dbt Cloud environment variable:
+
+<File name="packages.yml">
+
+```yaml
+packages:
+	- private: dbt-labs/awesome_repo
+		
+	- package: normal packages
+	
+	[...]
+```
+
+</File>
+
+You can pin private packages similar to regular dbt packages:
```

Comment:
i wasn't sure if the `revision` key was an array or what. what about adding something like this to help users (if that's right?):

You can pin private packages similar to regular dbt packages using the `revision` key. It specifies the version of the private package to retrieve. You can use a branch name, a tagged Git release, or a full 40-character commit hash:

Reply:
also does the key accept just one value?

Reply:
Yeah, it's pinned like regular packages. Updated the example